api_tests/src/actors/actor.ts

```diff
@@ -681,6 +673,44 @@ export class Actor {
         }
     }
 
+    public async createHerc20Halbit(
+        body: Herc20HalbitPayload
+    ): Promise<string> {
+        // @ts-ignore
```

Comment:
```suggestion
        // @ts-ignore: We ignore because a b c
```
An explanation is always better so that someone else can remove it if they fix the issue upstream.